proseco/guide.py

```diff
@@ -100,8 +100,19 @@ def thumbs_up(self):
             out = 1
         elif len(self) == 0:
             out = 0
+        # If this is an ER as inferred by number of requested guide stars, apply
+        # more requirements on the guide catalog to get a thumbs_up = 1.
+        elif self.n_guide > 6:
```

Comment:
It looks like I don't have a trivial way to impliment the higher P2 lim on thumbs_up in acq.py because I have no way of telling ERs from ORs.  For guide I think we can infer from the requested number of guide stars.